mta/fixtures/job_function.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from conftest import generate_string
+from swagger_client.models.api_job_function import ApiJobFunction
+
+
+@pytest.fixture(scope="session")
```

Comment:
@kpunwatk please change the scope to "function"

Reply:
Thanks @mguetta1 Done.